src/components/Modal/Modal.vue

```diff
@@ -26,15 +26,14 @@
 <template>
 	<div>
 		<button @click="showModal">Show Modal</button>
-		<modal v-if="modal" @close="closeModal">
+		<modal v-if="modal" @close="closeModal" size="prompt">
 			<div class="modal__content">Hello world</div>
 		</modal>
 	</div>
 </template>
 <style scoped>
```

Comment:
So, I am really not sure about this one, but maybe setting the lang to `scss` can help to use variables. I am not sure where the variable is defined in the first place.

```suggestion
<style scoped lang="scss">
```

Reply:
No, that doesn't seem to work unfortunatley. 
$header-size is set in the scss code of the modal here:
https://github.com/nextcloud/nextcloud-vue/blob/ab33f33ed82a71a72e53c9bbb83bcc27e7e9b6c2/src/components/Modal/Modal.vue#L532

Reply:
Maybe @skjnldsv can give use some insight on css/scss good practices

Reply:
The problem is that
```
height: calc(100% - $header-size);
```
Doesnt work...

Reply:
Since we have https://github.com/nextcloud/nextcloud-vue/blob/c5f9c2e57ce158546dc49cf5ef92394cd0ffaa32/webpack.common.js#L113 which imports the SCSS variables into all files I think moving https://github.com/nextcloud/nextcloud-vue/blob/c5f9c2e57ce158546dc49cf5ef92394cd0ffaa32/src/components/Modal/Modal.vue#L527
into https://github.com/nextcloud/nextcloud-vue/blob/c5f9c2e57ce158546dc49cf5ef92394cd0ffaa32/src/assets/variables.scss would make it work :)

Reply:
https://github.com/nextcloud/server/blob/4f90766ba314171bbfc78d1e988307c50633e7f3/core/css/css-variables.scss#L59

Reply:
Done, thanks for the pointer! :)

Reply:
Shall I also change this everywhere where $header-size is currently used? :)

Reply:
@skjnldsv $header-size is only used in the modal component so it can be easily changed.

Reply:
Yes, change it here then :)

Reply:
Not done yet

Reply:
should be done now